command/quota_apply_test.go

```diff
@@ -97,3 +99,49 @@ func TestQuotaApplyCommand_Good_JSON(t *testing.T) {
 	assert.Nil(t, err)
 	assert.Len(t, quotas, 1)
 }
+
+func TestQuotaApplyNetwork(t *testing.T) {
+	t.Parallel()
+	// srv, _, _ := testServer(t, true, nil)
+	// defer srv.Shutdown()
+
+	type example struct {
+		q *api.QuotaSpec
+		e string
+	}
+
+	intPtr := func(x int) *int {
+		return &x
+	}
+
+	cases := map[string]example{
+		`limit {region = "global", region_limit {network {mbits = 20}}}`: {
+			&api.QuotaSpec{
+				Limits: []*api.QuotaLimit{{
+					Region: "global",
+					RegionLimit: &api.Resources{
+						Networks: []*api.NetworkResource{{
+							MBits: intPtr(20),
+						}},
+					},
+				}},
+			},
+			"",
+		},
+		`limit {region = "global", region_limit {network { mbits = 20, device = "eth0"}}}`: {
+			nil,
+			"1 error(s) occurred:\n\n* limit -> region_limit -> resources -> network -> invalid key: device",
+		},
+	}
+
+	for hcl, expected := range cases {
+		q, e := parseQuotaSpec([]byte(hcl))
```

Comment:
I'd use `t.Run(hcl, func() { ... })` to make it obvious the case that's failing, should it fail ever.